libs/platform/src/lib/components/list/standard-list-item/standard-list-item.component.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { StandardListItemComponent } from './standard-list-item.component';
+import { PlatformListModule } from '../list.module';
+import { By } from '@angular/platform-browser';
+
+describe('StandardListItemComponent', () => {
```

Comment:
We should be testing all the properties of the StandardListItemComponent here. Things like setting `showNavigationArrow` to `true` and checking whether the class which shows the navigation arrow is set on the anchor tag.

Reply:
Done